### ISSUE TITLE
what is mel-spectrogram?

### ISSUE THREAD

**Issue by thsno02** [thsno02/StarGANv2-VC#1]
rt

**Comment by thsno02** [comment]
+ **audio signal**: the samples of air pressure over time to digitally
+ **[fast Fourier transformer](https://en.wikipedia.org/wiki/Fast_Fourier_transform)**: an algorithm that computes the discrete Fourier transform (DFT) of a sequence, or its inverse (IDFT)
+ **[spectrogram](https://en.wikipedia.org/wiki/Spectrogram)**: A spectrogram is a visual representation of the [spectrum](https://en.wikipedia.org/wiki/Spectral_density) of [frequencies](https://en.wikipedia.org/wiki/Frequencies) of a signal as it varies with time.
+ **mel spectrogram**: A mel spectrogram is a spectrogram where the frequencies are converted to the mel scale.

steps:

1. get the signals
2. implement fast Fourier transform to get the frequency
3. get the spectrogram, y-axis is in a log manner, color dimension is converted to decibels
4. convert to mel spectrogram

In a nutshell, mel-spectrogram is a spectrogram used to extract features of the voice frequencies.

REF1: https://medium.com/analytics-vidhya/understanding-the-mel-spectrogram-fca2afa2ce53
REF2: https://blog.csdn.net/weixin_50547200/article/details/117294164
